webgoat-lessons/jwt/src/main/resources/js/questions_jwt.json

```diff
@@ -0,0 +1,20 @@
+{
+  "questions": [
+    {
+      "text": "What is the result of the first code snippet?",
+      "solutions": {
+        "1": "Throws an exception in line 13",
```

Comment:
The line numbers are off by one, they should be 12, 7 and 9

Reply:
Fixed